internal/service/ec2/ipam_.go

```diff
@@ -89,6 +89,10 @@ func ResourceIPAM() *schema.Resource {
 				Type:     schema.TypeInt,
 				Computed: true,
 			},
+			"tier": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
```suggestion
				Optional: true,
				ValidateFunc: validation.StringInSlice(ec2.IpamTier_Values(), false),
```

Reply:
Thanks for the suggestions! Implemented the suggested changes  in 23e9e4f2e3bc5b6f0273eddb7c14fd878ecb8b6f